pkg/client/client.go

```diff
@@ -233,6 +262,19 @@ func NewClient(opts ...Option) (*Client, error) {
 		}
 	}
 
+	if client.indexFactory == nil {
+		packHome, err := iconfig.PackHome()
+		if err != nil {
+			return nil, errors.Wrap(err, "getting pack home")
+		}
+		indexRootStoragePath := filepath.Join(packHome, "manifests")
+
+		if xdgPath, ok := os.LookupEnv(xdgRuntimePath); ok {
+			indexRootStoragePath = xdgPath
+		}
```

Comment:
nit: we could move this to the top of the block to avoid getting the pack home when the env var is set